spd/experiments/ih/ih_config.yaml

```diff
@@ -29,7 +29,8 @@ recon_layerwise_coeff: null
 stochastic_recon_layerwise_coeff: 1
 importance_minimality_coeff: 1e-2
 pnorm: 0.1
-output_loss_type: kl
+output_recon_loss_type: kl
+hidden_act_recon_coeff: 0.0
```

Comment:
I want to rewrite our losses and evals so that we don't have to do this. If we want to track something but not compute it in our loss, we'd just run it in our evals and not on every training step. I'll be looking to make a PR to address this in the coming days. For now I guess it's OK to do this provided adding the hidden reconstruction calculation every step doesn't meaningfully increase runtime or memory consumption (e.g. by more than 5%).